src/runtime/rpc/network.cpp

```diff
@@ -281,7 +287,7 @@ void rpc_session::send_message(message_ex *msg)
         // Attention: here we only allow two cases to send message:
         //  case 1: session's state is SS_CONNECTED
         //  case 2: session is sending negotiation message
-        if ((SS_CONNECTED == _connect_state || security::is_negotiation_message(msg->rpc_code())) &&
+        if ((SS_CONNECTED == _connect_state || security::in_white_list(msg->rpc_code())) &&
```

Comment:
I suggest that you should update the comment, because a new case has been added.

Reply:
Yes, you are right. sorry I forgot it.